apps/src/code-studio/levels/postOnContinue.js

```diff
@@ -16,7 +19,14 @@ import {TestResults} from '@cdo/apps/constants';
  * Make our milestone post with some simple configuration. Note, this assumes
  * the level is a type that will always post "success".
  */
-export function postMilestoneForPageLoad() {
+export function onContinue() {
+  // prevent multiple milestone reports
+  var submitButton = $('.submitButton');
```

Comment:
Is there a reason you're not using `const` here?

Reply:
Heh, it was because I copied this from similar code somewhere else, will fix.